modules/images/webp-uploads/load.php

```diff
@@ -427,6 +427,20 @@ function webp_uploads_img_tag_update_mime_type( $image, $context, $attachment_id
 		return $image;
 	}
 
+	/**
+	 * Filters whether the smaller image should be used regardless of which MIME type is preferred overall.
+	 *
+	 * This is disabled by default only because it is not part of the current WordPress core feature proposal.
+	 *
+	 * By enabling this, the plugin will compare the image file sizes and prefer the smaller file regardless of MIME
+	 * type.
+	 *
+	 * @since n.e.x.t
```

Comment:
The filter already exists since 1.0.0, i.e. this shouldn't get `n.e.x.t`. You only moved the filter elsewhere, but it's not new.
```suggestion
	 * @since 1.0.0
```

Reply:
Thanks @felixarntz updated